windows/installer-source/KaliteSetupScript.iss

```diff
@@ -435,16 +448,16 @@ begin
     PipPath := GetPipPath;
     if PipPath = '' then
         exit;
-    PipCommand := 'install "' + ExpandConstant('{app}') + '\ka-lite\ka_lite_static-' + '{#TargetVersion}' + '-py2-none-any' + '.whl"';
+    PipCommand := 'install "' + ExpandConstant('{app}') + '\ka-lite\ka_lite-' + '{#TargetVersion}' + '-py2-none-any' + '.whl"';

```

Comment:
If it isn't using `ka-lite-static`, dependencies will be missing - I'm confused: Is this the correct .whl file with a different name?

Reply:
@benjaoming it should be `ka-lite-static` I pick the wrong `whl` file when building the installer.